src/extensions/askUserForConsent.ts

```diff
@@ -64,6 +65,33 @@ export async function displayRoles(
   utils.logLabeledBullet(logPrefix, message);
 }
 
+/**
+ * Displays APIs that will be enabled for the project and corresponding descriptions.
+ * @param extensionName name of extension to install/update
+ * @param projectId ID of user's project
+ * @param apis APIs that require user approval
+ */
+export async function displayApis(
+  extensionName: string,
+  projectId: string,
+  apis: extensionsApi.Api[]
+): Promise<void> {
+  if (!apis.length) {
+    return;
+  }
+  const question = `${clc.bold(
+    extensionName
+  )} will enable the following APIs for project ${clc.bold(projectId)}`;
+  const results: string[] = await Promise.all(
```

Comment:
Why is this `await Promise.all`? The code you have inside it is just constructing + returning a string atm

Reply:
LOL. I admit, I copy-pasted.... and then removed the async part... Removed.